lib/screens/login_screen.dart

```diff
@@ -0,0 +1,147 @@
+                  print('Not Verified user! ');
+                }
+
+                final email_ = email.text;
+                final password_ = password.text;
+                try {
+                  final UserCredential = await FirebaseAuth.instance
+                      .signInWithEmailAndPassword(
+                          email: email_, password: password_);
+                  print(UserCredential);
+                } on FirebaseAuthException catch (e) {
+                  print(e.runtimeType);
+
+                  if (e.code == 'user-not-found') {
+                    print(e.runtimeType);
+                    print('User not found !');
+                    // ignore: use_build_context_synchronously
+                    return showDialog(
+                        context: context,
+                        builder: ((context) {
+                          return Container(
+                            child: AlertDialog(
+                              title: const Text(
+                                  'You are not Registered yet, Register now !'),
+                              actions: [
+                                TextButton(
+                                  onPressed: () {
+                                    Navigator.of(context)
+                                        .pushNamedAndRemoveUntil(
+                                            registerRoute, (route) => false);
+                                  },
+                                  child: const Text('Ok'),
+                                )
+                              ],
+                            ),
+                          );
+                        }));
+                  } else if (e.code == 'wrong-password') {
+                    print(e.runtimeType);
+                    print('Entered wrong Password !');
+                    // ignore: use_build_context_synchronously
+                    return showDialog(
+                        context: context,
+                        builder: ((context) {
+                          return Container(
+                            child: AlertDialog(
+                              title: const Text(
+                                  'You entered wrong password. Try again !'),
+                              actions: [
+                                TextButton(
+                                  onPressed: () {
+                                    Navigator.of(context)
+                                        .pushNamedAndRemoveUntil(
+                                            loginRoute, (route) => false);
+                                  },
+                                  child: const Text('Ok'),
+                                )
+                              ],
+                            ),
+                          );
+                        }));
+                  }
+                  validator:
```

Comment:
don't use package for validating email or any thing, instead use TexrFormField and make your own validations